app/models/calculator/weight.rb

```diff
@@ -12,19 +12,5 @@ def compute(object)
       total_weight = line_items.sum { |li| ((li.variant.andand.weight || 0) * li.quantity) }
       total_weight * preferred_per_kg
     end
-
-    private
-
-    def line_items_for(object)
```

Comment:
Sorry @luisramos0, I didn't mean to just remove it. I have a similar memory to you that this was done for a reason. If I remember it correctly, it can happen that the weight calculator class is reloaded (maybe only in dev?) and it picks up the calculator class as parent without our override. It's missing the `line_items_for` implementation then. This is just a guess. I'm not sure what the actual problem was or if it even exists. One possible solution is to extract this code into a separate module that is included in the override and the weight calculator.

I'm just wondering if this change would actually be better in the master branch. What do you think?

Reply:
But if reloading in development caused a problem with the decorators, we would have noticed by now I think. We have tons of these, although yes, `class_eval` can have nasty side effects.

Reply:
I really dont feel like doing it now but the correct decision here is to do this in master first 😭

Reply:
I know it's a pain, but it would really be better to do this in master first. We have proper testing there. We don't want random orders to fail after the Spree upgrade, because the calculator isn't working properly. A new day and new motivation?

Reply:
ehehe, yes, will do this at some point. I move this PR to WIP for now.